pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/agent/impl/ContainerAgentProvider.java

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.pipeline.modeldefinition.agent.impl;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+public class ContainerAgentProvider {
```

Comment:
Definitely needs javadoc since there’s nothing actually in here to help figure it out.

Reply:
Ah, I see this is incomplete - this looks, from other code, to actually be an extension point?

Reply:
would "Provides ContainerAgent" javadoc be really helpful here ?
isn't class name obvious ?